metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script will measure network bandwidth with iperf3 tool using a
+# remote setup, where host A will run a server container and host B will
+# run a client container.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/remote-networking-test-common.sh"
+
+# Time to run the server using iperf3
+server_time=30
+
+# This function describes how to use this script
+function help {
+echo "$(cat << EOF
+Usage: $0 [-h] [--help]
```

Comment:
Haven't tried this, but should the help show? :
`Usage: $0 "[options]" "<interface_name>" "<user>" "<ip_address>"`

So that the user knows in which order he needs to add the arguments. 

I also agree with @MarioCarrilloA comment that we should use `getopt` for parsing the arguments, but I think none or most of the scripts do not use it, so maybe not a problem if not including it in this PR. Anyway I'll open an issue so this can be improve in the near future.

Reply:
If you want a getopt example then I just updated the launch_times metric from a fixed commandline to a '-x option' style getopt one - it is much nicer.
The code is in here: https://github.com/clearcontainers/tests/pull/676
And I stole it from @MarioCarrilloA fio test script ;-)